jinahub/encoders/text/AudioCLIPTextEncoder/README.md

```diff
@@ -0,0 +1,164 @@
+
+To do this, copy the `scripts/download_full.sh` script to your current directory (and make it executable):
+
+```
+wget https://raw.githubusercontent.com/jina-ai/executors/main/jinahub/encoders/text/AudioCLIPTextEncoder/scripts/download_full.sh && chmod +x download_full.sh
+./download_full.sh
+```
+
+This will download the `Full` version of the model (this is the default model used by the executor). If you instead want to download the `Partial` version of the model, execute
+
+```
+wget https://raw.githubusercontent.com/jina-ai/executors/main/jinahub/encoders/text/AudioCLIPTextEncoder/scripts/download_partial.sh && chmod +x download_partial.sh
+./download_partial.sh
+```
+
+And then you will also need to pass the argument `model_path='.cache/AudioCLIP-Partial-Training.pt'` when you initialize the executor.
+
+## 🚀 Usages
+
+### 🚚 Via JinaHub
+
+#### Using docker images
+
+Use the prebuilt images from JinaHub in your python code, 
+
+```python
+from jina import Flow
+	
+f = Flow().add(
+	uses='jinahub+docker://AudioCLIPTextEncoder',
+	volumes='/path/to/pwd/.cache:/workspace/.cache'
+)
+```
+
+or in the `.yml` config.
+	
+```yaml
+jtype: Flow
+pods:
+  - name: encoder
+    uses: 'jinahub+docker://AudioCLIPTextEncoder'
+    volumes: '/path/to/pwd/.cache:/workspace/.cache'
+```
+
+#### Using source codes
+
+Use the source code from JinaHub in your python code,
+
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub://AudioCLIPTextEncoder')
+```
+
+or in the `.yml` config.
+
+```yaml
+jtype: Flow
+pods:
+  - name: encoder
+    uses: 'jinahub://AudioCLIPTextEncoder'
+```
+
+<details>
+<summary>Click here to see advance usage</summary>
+
+### 📦️ Via pip
+
+1. Install the `jinahub-audioclip-text` package.
+
+	```bash
+	pip install git+https://github.com/jina-ai/executor-text-audioclip-text-encoder.git
+	```
+
+1. Use `jinahub-audioclip-text` in your code
+
+	```python
+	from jina import Flow
+	from jinahub.encoder.audioclip_text import AudioCLIPTextEncoder
+	
+	f = Flow().add(uses=AudioCLIPTextEncoder)
+	```
+
+
+### 🐳 Via Docker
+
+1. Clone the repo and build the docker image
+
+	```shell
+	git clone https://github.com/jina-ai/executor-text-audioclip-text-encoder.git
+	cd executor-text-audioclip-text-encoder
+	docker build -t jinahub-audioclip-text .
+	```
+
+1. Use `jinahub-audioclip-text` in your codes
+
+	```python
+	from jina import Flow
+	
+	f = Flow().add(
+		uses='docker://jinahub-audioclip-text:latest',
+		volumes='/path/to/pwd/.cache:/workspace/.cache'
+	)
+	```
+	
+</details>
+
+## 🎉️ Example 
+
+Here's a basic example demonstrating the use of this encoder
+
+```python
+from jina import Flow, Document
+
+f = Flow().add(
+	uses='jinahub+docker://AudioCLIPTextEncoder',
+	volumes='/path/to/pwd/.cache:/workspace/.cache'
+)
+
+with f:
+	doc = Document(text='test text')
```

Comment:
4 extra spaces here

Reply:
It just looks like that - it is actually a single tab. I will replace the tab with 4 spaces here then